.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
why ubuntu?

Reply:
This defines where the `nom publish` job as part of GitHub Actions runs on. 

Ubuntu was the default, so I kept it.